### ISSUE TITLE
Please update the implementation to use the graphic elements you used in Naiya's reels

### ISSUE THREAD

**Issue by drstarson** [Passiolife/Nutrition-AI-Reels#2]
- [x] Essentially we want it to be nicer. Specifically the panel view on the top with calories, protein, fat and carb icons

- [x] Keep both your current implementation and the implementation from Naiya where you're showing the food card

- [x] Show 3 options at the bottom allowing users to select 3 different reel modes

- [x] Show a sound option allowing users to select 3 sound options. @jryler  please provide tracks

**Comment by nikunj-MindInventory** [comment]
@drstarson

Few questions:

> Keep both your current implementation and the implementation from Naiya where you're showing the food card
1. Could you please explain in detail about this? Is this related to showing 3 options at the bottom allowing users to select  different reel modes?

> Show 3 options at the bottom allowing users to select 3 different reel modes
2. Could you please provide me 3 reels modes? 

> Show a sound option allowing users to select 3 sound options. @jryler please provide tracks
3. Dmitriy, It would be better if I could get a visually appealing design from Jeremy. So that we can ensure the demo's look and feel. 

**Suggestion on Design:** 
Dmitriy, We can provide a separate setting/selection screen that allows the user to select reel modes, camera or video mode, and soundtracks. Our camera screen will be less cluttered if we separate these UI controls from the camera screen, and reel recording will just record food recognition. CC: @jryler 


@jryler It'll be better if you suggest/provide a design for
- Allowing user to choose between different Reels options
- Allowing user to choose between different sounds options


@drstarson 
As the first point is clear to me, I'll begin with implementation. Other options are require clarification.

Thanks

**Comment by drstarson** [comment]
> Could you please explain in detail about this? Is this related to showing 3 options at the bottom allowing users to select different reel modes?

Yes. Essentially we want to give developers several examples of reels they can create.

**Comment by drstarson** [comment]
> Dmitriy, We can provide a separate setting/selection screen that allows the user to select reel modes, camera or video mode, and soundtracks. Our camera screen will be less cluttered if we separate these UI controls from the camera screen, and reel recording will just record food recognition. CC: @jryler

Yes. This is a good idea. For three modes we can have three buttons on a landing page. 

Always keep in mind that this is for developers. We want them to use your code to mix and match so o.k. to keep things separate.

**Comment by drstarson** [comment]
@nikunj-MindInventory 
Please move as far as you can and push the code by EOD Mon. Record your demo and provide updates. 

@jryler Let's review on Monday and provide feedback to Nikunj.

**Comment by nikunj-MindInventory** [comment]
@drstarson @jryler 

Check below video and let me know.

1. Plain
https://user-images.githubusercontent.com/89015620/233999890-ffbb8ea7-3e05-4dbd-9684-83754b45781f.MP4

2.  Food with Card
https://user-images.githubusercontent.com/89015620/233999826-0fdb12ea-5009-43e0-b738-cd5c9d316913.MP4

3. Settings screen:

https://user-images.githubusercontent.com/89015620/233999687-d8062464-8ba5-4503-9922-bfb3dfe3942a.MP4

**Comment by drstarson** [comment]
@nikunj-MindInventory it looks mostly good from the function perspective. But it looks very bad from presentability. Can you make the buttons look nicer like they do in Naiya? 

Please push the code and please record a few demos using real fruit and foods.

**Comment by nikunj-MindInventory** [comment]
Design update: https://www.figma.com/file/RqpRK7K7MLAuU8jmTGhHUC/Reel-Mode?node-id=0%3A1&t=4bviLJ1KdH2RbMH1-1

CC: @jryler @drstarson

**Comment by nikunj-MindInventory** [comment]
Design Updated Video for the same:

1. Video:
https://user-images.githubusercontent.com/89015620/234574264-521bf15f-ac41-4f85-9946-f9c14520bfaa.MP4

2. Settings:
![Settings](https://user-images.githubusercontent.com/89015620/234574368-d2ae4b6c-6b81-4c4d-829d-22668e41a5e0.PNG)
